core/domain/base_model_validators.py

```diff
@@ -84,10 +86,23 @@ def __init__(
             model_class: ClassObject. The external model class.
             model_ids: list(str). The list of external model ids to fetch the
                 external models.
-        """
+            remove_system_user_ids: bool. Whether to remove the system user IDs
```

Comment:
Why the fetcher object will remove the user_ids?

Reply:
Migration bot and pseudo user ids do not have user setting model. Thus errors are being caused in the validation methods. Thus removing them will avoid these errors. These are removed in the fetcher because these can't be fetched.